src/main/java/org/breedinginsight/services/parsers/trait/TraitFileParser.java

```diff
@@ -125,7 +125,7 @@ public TraitFileParser(TraitFileValidatorError traitValidatorError){
             ExcelRecord record = records.get(i);
 
             ProgramObservationLevel level = ProgramObservationLevel.builder()
-                    .name(parseExcelValueAsString(record, TraitFileColumns.TRAIT_LEVEL))
+                    .name(parseExcelValueAsString(record, TraitFileColumns.TRAIT_ENTITY))
```

Comment:
Is entity really the same as observation level or is it just being used for the autocomplete features of the program observation levels?

Reply:
just for the autocomplete features.
The card indirectly states that the two are not the same by saying that obs level should not be associated with traits.